src/rpc/blockchain.cpp

```diff
@@ -753,12 +753,12 @@ UniValue getblock(const JSONRPCRequest& request)
             verbosity = request.params[1].get_bool() ? 1 : 0;
     }
 
-    if (mapBlockIndex.count(hash) == 0)
+    CBlockIndex* pblockindex = LookupBlockIndex(hash);
```

Comment:
Another one that could be const. There are more, but I'll stop commenting every one.